test/GitcoinGovernor.t.sol

```diff
@@ -427,3 +428,466 @@ contract GitcoinGovernorAlphaPostProposalTest is GitcoinGovernorProposalTestHelp
     governorAlpha.queue(_newProposalId);
   }
 }
+
+contract NewGitcoinGovernorProposalTest is GitcoinGovernorProposalTestHelper {
+  // From GovernorCountingSimple
+  uint8 constant AGAINST = 0;
+  uint8 constant FOR = 1;
+  uint8 constant ABSTAIN = 2;
+
+  function assumeReceiver(address _receiver) public {
+    vm.assume(_receiver != TIMELOCK && _receiver != address(0x0));
+  }
+
+  function buildProposalData(string memory _signature, bytes memory _calldata)
+    public
+    pure
+    returns (bytes memory)
+  {
+    return abi.encodePacked(bytes4(keccak256(bytes(_signature))), _calldata);
+  }
+
+  function jumpToActiveProposal(uint256 _proposalId) public {
+    uint256 _snapshot = governorBravo.proposalSnapshot(_proposalId);
+    vm.roll(_snapshot + 1);
+  }
+
+  function jumpToVotingComplete(uint256 _proposalId) public {
+    // Jump one block past the proposal voting deadline
+    uint256 _deadline = governorBravo.proposalDeadline(_proposalId);
+    vm.roll(_deadline + 1);
+  }
+
+  function jumpPastProposalEta(uint256 _proposalId) public {
+    uint256 _eta = governorBravo.proposalEta(_proposalId);
+    vm.roll(block.number + 1);
+    vm.warp(_eta + 1);
+  }
+
+  function delegatesVoteOnBravoGovernor(uint256 _proposalId, uint8 _support) public {
+    assertLt(_support, 3, "Invalid value for support");
+
+    for (uint256 _index = 0; _index < delegates.length; _index++) {
+      vm.prank(delegates[_index]);
+      governorBravo.castVote(_proposalId, _support);
+    }
+  }
+
+  function submitTokenSendProposal(address _token, uint256 _amount, address _receiver)
+    public
+    returns (uint256, address[] memory, uint256[] memory, bytes[] memory, string memory)
+  {
+    // Craft a new proposal to send GTC
+    address[] memory _targets = new address[](1);
+    uint256[] memory _values = new uint256[](1);
+    bytes[] memory _calldatas = new bytes[](1);
+
+    _targets[0] = _token;
+    _values[0] = 0;
+    _calldatas[0] = buildProposalData("transfer(address,uint256)", abi.encode(_receiver, _amount));
+    string memory _description = "Transfer some tokens from the new Governor";
+
+    // Submit the new proposal
+    vm.prank(PROPOSER);
+    uint256 _newProposalId = governorBravo.propose(_targets, _values, _calldatas, _description);
+
+    return (_newProposalId, _targets, _values, _calldatas, _description);
+  }
+
+  function assertEq(IGovernor.ProposalState _actual, IGovernor.ProposalState _expected) public {
+    assertEq(uint8(_actual), uint8(_expected));
+  }
+
+  function setUp() public override {
+    GitcoinGovernorProposalTestHelper.setUp();
+  }
```

Comment:
Nit, can delete this since you don't do anything extra here, the `GitcoinGovernorProposalTestHelper` setUp will still be called. Feel free to resolve this comment yourself.

Reply:
Done in 8d1ac76